client/src/components/SocialShare.tsx

```diff
@@ -46,6 +45,37 @@ type SocialShareProps = {
   customContent?: SocialShareContent;
 };
 
+const _FacebookButton: React.FC = (props) => (
+  <Button
+    labelText="Facebook (Opens a new window)"
+    variant="secondary"
+    size="small"
+    labelIcon={IconPerson}
+    iconOnly
+    {...props}
+  />
+);
+const _TwitterButton: React.FC = (props) => (
+  <Button
+    labelText="Twitter (Opens a new window)"
+    variant="secondary"
+    size="small"
+    labelIcon={IconTwitter}
+    iconOnly
+    {...props}
+  />
+);
+const _EmailButton: React.FC = (props) => (
+  <Button
+    labelText={`${i18n._("Email")} (Opens a new window)`}
```

Comment:
If you could use this `i18n._("Email")` format for all of the label text for these three that will ensure the translations work.

Reply:
For sure! Do I need to run anything to get it into the dictionary? Or do you want to handle that later?

Reply:
I think we're going to run the localization script for the whole feature branch all at once when we're ready for prod. but if you ever do need it -- the script is `yarn lingui extract`

Reply:
Thanks, good to know!